tests/__snapshots__/test_help_messages.ambr

```diff
@@ -1,30 +1,4 @@
 # serializer version: 1
-# name: test_help_messages[app.bundle]
-  '''
-                                                                                  
-   Usage: default app bundle [OPTIONS]                                            
```

Comment:
Is this expected change? Looks like it is not related to image-registry login

Reply:
Good point, I've retargeted this pull request on a separate branch that gets rid of the duplicate snapshots.